roles/tackle/defaults/main.yml

```diff
@@ -161,6 +161,24 @@ analyzer_container_limits_memory: "4Gi"
 analyzer_container_requests_cpu: "1"
 analyzer_container_requests_memory: "4Gi"
 
+provider_generic_component_name: "addon"
+provider_generic_container_limits_cpu: "1"
+provider_generic_container_limits_memory: "4Gi"
```

Comment:
I don't think the generic provider needs 4Gi.  @pranavgaikwad can you comment?

Reply:
true, should be good at 1Gi